analyzers/src/SonarAnalyzer.SourceGenerators/SonarAnalyzer.SourceGenerators.csproj

```diff
@@ -5,9 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.1.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" PrivateAssets="all" />
```

Comment:
This is one version below the latest release 4.5.0, because 4.5.0 requires VS 17.5.0 and (silently) doesn't work without that.
We need to wait some time before bumping it.